Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +12,7 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array?.Length > 0) Array.Sort(array);
```

Comment:
Hah! Please do not cheat, before you can use these functinos, you need to understand what happens under the hood, please implement it yourself.